lib/src/visitor/async_evaluate.dart

```diff
@@ -1481,6 +1481,13 @@ class _EvaluateVisitor
         return;
       }
 
+      // If only built-in modules are loaded, we still need a separate
+      // environment to ensure their namespaces aren't exposed in the outer
+      // environment, but we don't need to worry about `@extend`s.
```

Comment:
Explain what not having to worry about extends means concretely. Maybe something like "...so we can add styles directly to the existing stylesheet rather than creating a new one"?